app/printing/printFull/get.controller.js

```diff
@@ -26,10 +28,26 @@ const printFullSentencePlan = async ({ path, params: { id, planId }, tokens }, r
       return res.render('app/error', { error })
     }
 
+    // only get intervention data if there is any objective with an action with an intervention
+    const hasInterventions = objectives.some(objective => {
+      return objective.actions.some(({ intervention }) => intervention)
+    })
+
+    let interventionList = []
+    if (hasInterventions) {
+      interventionList = await getInterventions(tokens)
+    }
```

Comment:
Probably overcomplicated, but this could be done with a ternary,then we get a bit of security from the const too:
```const interventionList = hasInterventions ? await getInterventions(tokens) : []```